src/main/java/views/ApplicationController/index.ftl.html

```diff
@@ -22,6 +22,12 @@ <h2 id="player_prompt">Place your Ships</h2>
 <div id="visibleButtons">
     <button id="new_game">New Game</button>
     <button id="surrender">Surrender</button>
+    <div id="movement_buttons" class="hidden">
```

Comment:
I would like to see this interface spruced up a bit, but our UI prototypes did not have considerations for a bunch more buttons. If we were going forward, we'd definitely come up with some new design ideas.